examples/all-clusters-app/all-clusters-common/src/dishwasher-mode.cpp

```diff
@@ -76,12 +76,24 @@ CHIP_ERROR DishwasherModeDelegate::GetModeTagsByIndex(uint8_t modeIndex, List<Mo
     return CHIP_NO_ERROR;
 }
 
+void DishwasherMode::Shutdown()
+{
+    if (gDishwasherModeInstance != nullptr)
+    {
+        delete gDishwasherModeInstance;
+        gDishwasherModeInstance = nullptr;
+    }
+    if (gDishwasherModeDelegate != nullptr)
+    {
+        gDishwasherModeDelegate->~DishwasherModeDelegate();
```

Comment:
Why is destructor explicitly called here?

Reply:
@hicklin This is .. strange.  This was one of my review comments, which was not addressed.  See https://github.com/project-chip/connectedhomeip/pull/28163#discussion_r1272573621 the "similar elsewhere" bit!

Reply:
Sorry about that. For some reason I didn't clock the delegate and only did this for the instance. I have issued a [bug](https://github.com/project-chip/connectedhomeip/issues/28265) and will be submitting a PR that fixes it soon.